Sources/ComposableArchitecture/Store.swift

```diff
@@ -368,8 +373,8 @@ public final class Store<State, Action> {
     self.publisherScope(state: toLocalState, action: { $0 })
   }
 
-  func send(_ action: Action, isFromViewStore: Bool = true) {
-    self.threadCheck(status: .send(action, isFromViewStore: isFromViewStore))
+  func send(_ action: Action, originatingFrom originatingAction: Action?) {
+    self.threadCheck(status: .send(action, originatingAction: originatingAction))
```

Comment:
This is unrelated to the purpose of this PR, but by threading the originating action for effect emissions we can provide better messaging when the thread check fails.